pandas/plotting/_core.py

```diff
@@ -1494,6 +1499,7 @@ def _args_adjust(self):
 
     @classmethod
     def _plot(cls, ax, y, column_num=None, return_type='axes', **kwds):
+        from pandas.core.series import remove_na
```

Comment:
I would be happy to move ``remove_na`` to ``pandas.core.dtypes.missing``, and rename to ``remote_na_arraylike`` (will have to update a couple of references)

```
(pandas) bash-3.2$ grep -R remove_na pandas
pandas/core/series.py:            result = remove_na(self)
pandas/core/series.py:def remove_na(series):
pandas/plotting/_core.py:from pandas.core.series import Series, remove_na
pandas/plotting/_core.py:        y = remove_na(y)
pandas/plotting/_core.py:            y = [remove_na(v) for v in y]
pandas/plotting/_core.py:            y = remove_na(y)
pandas/plotting/_core.py:        values = [remove_na(v) for v in values]
pandas/tests/test_panel.py:from pandas.core.series import remove_na
pandas/tests/test_panel.py:                nona = remove_na(x)
pandas/tests/test_panel4d.py:from pandas.core.series import remove_na
pandas/tests/test_panel4d.py:                nona = remove_na(x)
```